Tasks/3 - Condition/condition.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head></head>
+  <body>
+    <script>
+      var answer = prompt("What do you want to eat?");
```

Comment:
You can also use `const` for this variable

Reply:
done